lambda/src/main/scala/pricemigrationengine/handlers/NotificationHandler.scala

```diff
@@ -14,10 +14,15 @@ object NotificationHandler extends CohortHandler {
 
   val Successful = 1
   val Unsuccessful = 0
-
   val Cancelled_Status = "Cancelled"
 
+  // We are starting the notification process for any item whose start date is less than 49 (NotificationLeadTimeDays)
+  // days away, but because it can happen, we also need to react to items which for *any* reason have
+  // not been processed on time (meaning early enough) and for which the previously computed startDate is
+  // too close (less than NotificationPaddingTimeDays away) from now and would not give enough time to the letters.
+
   private val NotificationLeadTimeDays = 49
+  private val NotificationPaddingTimeDays = 35
```

Comment:
Perhaps these should be called something more like `StandardNotificationLeadTime` and  `minNotificationLeadTime` and given a duration value.  I don't know if that's any clearer but padding seems difficult to understand to me.